gunicorn/util.py

```diff
@@ -155,10 +154,6 @@ def set_owner_process(uid, gid, initgroups=False):
             except KeyError:
                 initgroups = False
 
-        # versions of python < 2.6.2 don't manage unsigned int for
-        # groups like on osx or fedora
-        gid = abs(gid) & 0x7FFFFFFF
```

Comment:
has this change been checked? fedora do some odd things sometimes...

Reply:
I've not checked it so have reverted it to be on the safe side.